packages/devkit/src/utils/convert-nx-executor.ts

```diff
@@ -17,18 +17,40 @@ export function convertNxExecutor(executor: Executor) {
       ? readNxJsonFromDisk(builderContext.workspaceRoot)
       : // TODO(v18): remove readNxJson. This is to be backwards compatible with Nx 16.5 and below.
         (workspaces as any).readNxJson();
-    const projectsConfigurations = workspaces.readProjectsConfigurations({
-      _includeProjectsFromAngularJson: true,
-    });
 
     const promise = async () => {
+      const nx = requireNx();
+      const projectsConfigurations =
+        nx.retrieveProjectConfigurations &&
+        nx.loadNxPlugins &&
+        nx.getNxRequirePaths
```

Comment:
We can remove this check in Nx 18, at which point devkit@18 will support Nx@17->Nx@19 which should all have these methods